osidb/tests/factories.py

```diff
@@ -49,6 +49,15 @@ class Meta:
         yes_declaration=factory.Faker("future_datetime", tzinfo=UTC),
         no_declaration=factory.Faker("past_datetime", tzinfo=UTC),
     )
+    summary = factory.Maybe(
+        "embargoed",
+        yes_declaration=factory.sequence(
+            lambda n: f"EMBARGOED - CVE-2022-000{n} kernel-rt: kernel: race condition in example function [rhel-1.0.0.z]"
```

Comment:
nitpick: the correct flaw title (though we do not have all the checks implemented yet) looks a bit differently (https://git.prodsec.redhat.com/devops/sfm2/-/blob/master/sfm2/helpers.py#L22) - it does not contain dash between EMBARGOED and the CVE(s) and there is no square bracket at the end (unlike for the tracker title) - though the square bracket would not matter to the validations as the end part is a free text

Reply:
refactored to reflect the right format